engineering/software-engineer-product.md

```diff
@@ -0,0 +1,73 @@
+---
+authors: @vipul-sharma20
+groups: product
+teams: runtime, platform
+level: L1, L2, L3
+active: true
+---
+
+# Software Engineer, Product
+
+We are looking for a Software Engineer in our Runtime Engineering team. This
+role will place you in the team which ensures the smooth running of our voice
+bot in our production systems so our callers have the smoothest experience.
+
+This means dealing heavily with optimisation, reliability, profiling and
+working with a myriad of teams within the company. You will be interfacing with
+product managers, ML engineers and delivery teams as a part of this role.
+
+With the knowledge of building scalable applications, fault tolerant systems
+and emerging technologies, you will be responsible for designing and managing
+applications, evolving and boosting productivity for the Skit Runtime
+Engineering team.
+
+You will be delivering a cutting edge product aimed at improving customer
+experience with Call Centres. You will be joining us at a time when we're on a
+shoot-the-moon style growth trajectory having grown our workforce 5x in the
+last year.
+
+### Responsibilities
+
+- Design and develop highly scalable, highly available, reliable, secure and
+  fault-tolerant systems.
+- Building and managing automated build/test/deployment environments.
+- Build and research new tools and frameworks for internal use.
+- Willingness to cross-team/role boundaries and work with other teams/other
```

Comment:
This doesn't look like responsiblity. Also a few pointers below.

Reply:
Agreed, these points are around the work style of this role and not responsibilities which is already covered in the role description in the previous section.